jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
```

Comment:
`["/src"]` will be sufficient

Reply:
also add `testRegex`
`  "testRegex": ".*\\.test.tsx?$"`

Reply:
Didn't add `testRegex` because the [default ](https://jestjs.io/docs/en/configuration#testregex-string--arraystring) value already matches the pattern of test/spec: `(/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$`